packages/transport/src/modules/sync/ExternalEventsProcessor.ts

```diff
@@ -32,16 +37,16 @@ export class ExternalEventsProcessor {
             try {
                 switch (externalEvent.type) {
```

Comment:
wtb building an `ExternalEventProcessorRegistry` that handles this switch..case for us?

Reply:
I don't see the benefit as the constructors are different because of the controllers. And the switch case is the only "logic" in this file